### ISSUE TITLE
bug: popover is not a known attribute of [element]

### ISSUE THREAD

**Issue by gfellerph** [ionic-team/stencil#4745]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/stencil/issues) that already report this problem, without success.

### Stencil Version

3.4.2

### Current Behavior

When adding an element with the popover attribute `<div popover></div>`, there is a typescript error on the console.

```shell
Type '{ popover: true; }' is not assignable to type
'HTMLAttributes<HTMLDivElement>'.Property 'popover' does not exist
on type 'HTMLAttributes<HTMLDivElement>'.
     <Host data-version={version}>
       <div popover></div>
       <div class="tabs-wrapper">
rebuild failed, watching for changes... in 106 ms
```

### Expected Behavior

The standard popover attribute is a known attribute.

### System Info

```shell
System: node 16.20.0
    Platform: darwin (22.5.0)
   CPU Model: Apple M1 Pro (8 cpus)
    Compiler: /Users/.../design-system/node_modules/.pnpm/@stencil+core@3.4.2/node_modules/@stencil/core/compiler/stencil.js
       Build: 1690208000
     Stencil: 3.4.2 ☀️
  TypeScript: 5.0.4
      Rollup: 2.42.3
      Parse5: 7.1.2
      Sizzle: 2.42.3
      Terser: 5.18.1
```

### Steps to Reproduce

Write a component that uses `popover` on any element.

### Code Reproduction URL

https://github.com/gfellerph/stencil-popover-repro

**Comment by rwaskiewicz** [comment]
Thanks @gfellerph! I've confirmed that this is an issue in the typings - we'll keep an eye on `popover`  (which is still marked as [experimental/in technical preview in Firefox and Safari](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/popover#browser_compatibility)) and add it accordingly. Thanks again!

**Comment by alicewriteswrongs** [comment]
The fix for this issue has been released as a part of today's [Stencil v4.8.0 release](https://github.com/ionic-team/stencil/releases/tag/v4.8.0)!
